tests/infrastructure/src/main/java/org/mule/test/infrastructure/process/Controller.java

```diff
@@ -68,7 +71,8 @@ public Controller(String muleHome, int timeout) {
     this.muleBin = getMuleBin();
     this.domainsDir = new File(muleHome + "/domains");
     this.appsDir = new File(muleHome + "/apps/");
-    this.libsDir = new File(muleHome + "/lib/user");
+    this.libsDir = new File(muleHome + LIB_USER_FOLDER);
+    this.pluginsDir = new File(muleHome + PLUGINS_FOLDER);
```

Comment:
What is the purpose of this change if plugins live inside applications and not in muleHome/plugins?